### DISCUSSION TITLE
Error in Public Testcase Walkthrough Step 6

### DISCUSSION THREAD

**Discussion by evanr1232** [NASA-LIS/LISF#1297] [Q&A]
Hello, I am currently trying to run LIS (./LIS -f lis.config_noah36_smapda), using the standard files provided in the test cases. I get the following error immediately: 

Abort(1) on node 0 (rank 0 in comm 496): application called MPI_Abort(comm=0x84000002, 1) - process 0

Additionally, this is the end of the log/lislog_smapda.0000 file:

[WARN] ****************WARNING*********************
 [WARN] Error in nf90_get_att: GRIDTYPE
 [WARN] ****************WARNING*********************
 [INFO] local obs domain:(          28          22 )
 [INFO] local obs domain without halo:(          28          22 )
 [INFO] running obs domain:(          28          22 )
 [INFO] local obs domain    34.37500       39.62500      -102.8750
  -96.12500     d:            1
 [INFO] --------------------Obs domain            1 ----------------------
 [INFO] (           1 ,  0.0000000E+00 )
 [INFO] (           2 ,   28.00000     )
 [INFO] (           3 ,   22.00000     )
 [INFO] (           4 ,   34.37500     )
 [INFO] (           5 ,  -102.8750     )
 [INFO] (           6 ,   128.0000     )
 [INFO] (           7 ,   39.62500     )
 [INFO] (           8 ,  -96.12500     )
 [INFO] (           9 ,  0.2500000     )
 [INFO] (          10 ,  0.2500000     )
 [INFO] (          11 ,   64.00000     )
 [INFO] (          12 ,  0.0000000E+00 )
 [INFO] (          13 ,  0.0000000E+00 )
 [INFO] --------------------------------------------------------------
 [INFO] Reading obs landmask map from ./lis_input.nldas.noah36.d01.nc
 [INFO] Successfully read obs landmask map
 [ERR] ESMF_DistGridCreate failed: obsgridDG Stopping.
 [ERR] endrun is being called

I would appreciate any help in solving this issue. Thanks!

**Comment by dmocko** [comment]
Hi @evanr1232 

That warning message at the top occurs for us as well.  We need to take of that separate issue.

Your problem is the ESMF error at the bottom of your log file.

From your previous Discussion message, it seems that you are running on NCCS's Discover.  Is that correct?

If so, would you mind sharing the location of your configure.lis file as well as your job script?

Please ensure that read permissions are open.

If you are not on Discover, please attach these files.

**Reply by evanr1232** [reply]
Hi @dmocko 

Yes, I am running on NCCS Discover.

The files are in /discover/nobackup/erobert8/lis-public-testcases. Let me know if you are unable to access them.

Thank you!

**Reply by evanr1232** [reply]
Hi @dmocko , 


I just restarted the test cases from scratch, starting at Test Case 1, and am still running into the same error when I reach Test Case 6. Please let me know if you have any ideas of how to fix this issue.

Thank you!

**Reply by dmocko** [reply]
Hi @evanr1232 

My apologies, I was unable to access files in your nobackup directory (I still am unable), but never contacted you to open them up.

Can you please do so, so I can take a look?
